misc/graylog.conf

```diff
@@ -131,7 +131,7 @@ rest_listen_uri = http://127.0.0.1:9000/api/
 # for the application.context configuration parameter in pre-2.0 versions of the Graylog web interface.
 #web_listen_uri = http://127.0.0.1:9000/
 
-# Web interface endpoint URI. This setting can be overriden on a per-request basis with the X-Graylog-Server-URL header.
+# Web interface REST endpoint URI. Used by browsers to receive steams. This setting can be overriden on a per-request basis with the X-Graylog-Server-URL header.
```

Comment:
It is not only `Used by browsers to receive steams.`, it does way more. I would recommend changing this to something like `Must be reachable from the user's browser.`

Reply:
Good point. I committed the change.